code/__defines/access_defs.dm

```diff
@@ -0,0 +1,193 @@
+#define ACCESS_REGION_NONE          -1
+#define ACCESS_REGION_ALL           0
+#define ACCESS_REGION_SECURITY      1
+#define ACCESS_REGION_MEDBAY        2
+#define ACCESS_REGION_RESEARCH      3
+#define ACCESS_REGION_ENGINEERING   4
+#define ACCESS_REGION_COMMAND       5
+#define ACCESS_REGION_GENERAL       6
+#define ACCESS_REGION_SUPPLY        7
+#define ACCESS_REGION_SERVICE       8
+
+#define ACCESS_REGION_CONTAINMENT	9
+#define ACCESS_REGION_OFFSITE		10
+
+// Keep those two up to date
```

Comment:
This was not kept up to date

Reply:
these are used only in two places; airlock electronics and access decrypter to determine regions
containment and offsite dont really fit there so i've decided to not keep it up to date (read: lazy)